src/file-uploader.scss

```diff
@@ -0,0 +1,31 @@
+@import "./mixins";
+@import "./new-settings";
+
+$block: #{$fd-namespace}-file-uploader;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: inline-flex;
+  align-items: center;
+  width: auto;
+
+  &__input.#{$fd-namespace}-input {
+    width: auto;
+    margin-right: 0.25rem;
```

Comment:
keep 0.25rem in a variable and reuse it at lines 15 and 18

Reply:
added variable